.ci/test

```diff
@@ -34,7 +34,7 @@ fi
 cd "${SOURCE_PATH}"
 
 # Build the Ginkgo (test framework) binary to be able to execute the tests.
-GO111MODULE=on go install -mod=vendor ./vendor/github.com/onsi/ginkgo/ginkgo
+GO111MODULE=on GOFLAGS="-mod=vendor" go install ./vendor/github.com/onsi/ginkgo/ginkgo
```

Comment:
Now half of the occurrences use `GOFLAGS="-mod=vendor"`, and the other half - `-mod=vendor`. Let's pick one and use only it.

Reply:
+1, please make it consistent. Let's wait with the merge then.